### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'discard'

### ISSUE THREAD

**Issue by sachinrekhi** [andrewgodwin/channels#104]
I've installed channels via PIP, currently running version 0.10.2. When running the dev server, after a little while I always see the following error in the dev server logs:
**AttributeError: 'dict' object has no attribute 'discard'**

I'm currently using the inmemory channel layer in dev. Given the stack trace looks like this is coming from the framework itself, it looks like the issue is not my own code but the framework itself. Stack trace below.

Please let me know how I can resolve this issue, thanks!

```
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/Users/sachin/Documents/Code/virtualenvs/notejoy/lib/python2.7/site-packages/channels/management/commands/runserver.py", line 152, in run
    worker.run()
  File "/Users/sachin/Documents/Code/virtualenvs/notejoy/lib/python2.7/site-packages/channels/worker.py", line 50, in run
    channel, content = self.channel_layer.receive_many(channels, block=True)
  File "/Users/sachin/Documents/Code/virtualenvs/notejoy/lib/python2.7/site-packages/asgiref/inmemory.py", line 51, in receive_many
    self._clean_expired()
  File "/Users/sachin/Documents/Code/virtualenvs/notejoy/lib/python2.7/site-packages/asgiref/inmemory.py", line 128, in _clean_expired
    self._remove_from_groups(channel)
  File "/Users/sachin/Documents/Code/virtualenvs/notejoy/lib/python2.7/site-packages/asgiref/inmemory.py", line 145, in _remove_from_groups
    channels.discard(channel)
AttributeError: 'dict' object has no attribute 'discard'
```

**Comment by sachinrekhi** [comment]
These errors do go away when I switch to the asgi_redis channel layer, so looks like the issue is specifically with the inmemory channel layer.

**Comment by andrewgodwin** [comment]
Yep, this is an error I introduced in 0.11 it looks like. Fixed and released as `asgiref 0.11.2`.
